packages/edit-site/src/components/style-book/index.js

```diff
@@ -222,7 +222,11 @@ function StyleBook( {
 	);
 
 	const settings = useMemo(
-		() => ( { ...originalSettings, __unstableIsPreviewMode: true } ),
+		() => ( {
+			...originalSettings,
+			focusMode: false, // Disable "Spotlight mode".
+			__unstableIsPreviewMode: true,
+		} ),
```

Comment:
I think this change in the `StyleBook` component isn't required; these `settings` aren't passed into `BlockEditorProvider`.

Side note: The component doesn't really need the complete set of settings. They're passed to `StyleBookBody`, which only uses `settings.styles`. But this is unrelated to the fix here.

cc @andrewserong, @ramonjd